io/io/src/TFile.cxx

```diff
@@ -396,6 +396,8 @@ TFile::TFile(const char *fname1, Option_t *option, const char *ftitle, Int_t com
    Bool_t devnull = kFALSE;
 
    if (!fname1 || !fname1[0]) {
+      strstr(fname1, "://") ?
```

Comment:
Can we use a simple, plain `if` here, please?

Reply:
Ok. Better to keep everything uniform